parser/parser.y

```diff
@@ -3606,6 +3611,10 @@ FunctionCallNonKeyword:
 			Args: []ast.ExprNode{ast.NewValueExpr($3), $5},
 		}
 	}
+|	"LOG" '(' ExpressionListOpt ')'
```

Comment:
Why add this?

Reply:
After this PR, "LOG" change to a token from an identifier, it's a not keyword token.
There is no rule for builtin function "select log(...)" , so I add this rule.